jobQueue/setup.js

```diff
@@ -18,19 +18,40 @@ const conf = require('../config');
 const featureToggles = require('feature-toggles');
 const urlParser = require('url');
 const kue = require('kue');
+const activityLogUtil = require('../utils/activityLog');
+
 const redisOptions = {
   redis: conf.redis.instanceUrl.queue,
 };
+
 const redisInfo = urlParser.parse(redisOptions.redis, true);
 if (redisInfo.protocol !== PROTOCOL_PREFIX) {
   redisOptions.redis = 'redis:' + redisOptions.redis;
 }
 
 const jobQueue = kue.createQueue(redisOptions);
 
+/**
+ * Kue's Queue graceful shutdown.
+ */
+function gracefulShutdown() {
+  const start = Date.now();
+  jobQueue.shutdown(conf.KUE_SHUTDOWN_TIME_OUT, (err) => {
+    if (featureToggles.isFeatureEnabled('enableSigtermActivityLog')) {
+      const status = 'Job queue shutdown: ' + (err || 'OK');
+      const logWrapper = {
+        status,
```

Comment:
Since the status string allows spaces and special characters, please wrap the string in double quotes.

Reply:
Done